File: FestivalJava/Networking/src/main/java/dto/ShowDTO.java
package dto;

import java.io.Serializable;
import java.util.Date;

/**
 * Created by Sebi on 29-Mar-17.
 */
public class ShowDTO implements Serializable {
    private Integer id;
    private String location;
    private Date date;
    private Integer ticketsAvailable;
    private Integer ticketsSold;
    private ArtistDTO artist;

    public ShowDTO(Integer id, String location, Date date,
                   Integer ticketsAvailable, Integer ticketsSold,
                   ArtistDTO artist){
        this.id = id;
        this.location = location;
        this.date = date;
        this.ticketsAvailable = ticketsAvailable;
        this.ticketsSold = ticketsSold;
        this.artist = artist;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getLocation() {
        return location;
    }

    public void setLocation(String location) {
        this.location = location;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public Integer getTicketsAvailable() {
        return ticketsAvailable;
    }

    public void setTicketsAvailable(Integer ticketsAvailable) {
        this.ticketsAvailable = ticketsAvailable;
    }

    public Integer getTicketsSold() {
        return ticketsSold;
    }

    public void setTicketsSold(Integer ticketsSold) {
        this.ticketsSold = ticketsSold;
    }

    public ArtistDTO getArtist() {
        return artist;
    }

    public void setArtist(ArtistDTO artist) {
        this.artist = artist;
    }
}

File: FestivalC#/Festival/Networking/dto/ShowDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Networking.dto
{
    [Serializable]
    public class ShowDTO
    {
        private int id;
        private string location;
        private DateTime date;
        private int ticketsAvailable;
        private int ticketsSold;
        private ArtistDTO artist;

        public int Id
        {
            get
            {
                return id;
            }

            set
            {
                id = value;
            }
        }

        public string Location
        {
            get
            {
                return location;
            }

            set
            {
                location = value;
            }
        }

        public DateTime Date
        {
            get
            {
                return date;
            }

            set
            {
                date = value;
            }
        }

        public int TicketsAvailable
        {
            get
            {
                return ticketsAvailable;
            }

            set
            {
                ticketsAvailable = value;
            }
        }

        public int TicketsSold
        {
            get
            {
                return ticketsSold;
            }

            set
            {
                ticketsSold = value;
            }
        }

        public ArtistDTO Artist
        {
            get
            {
                return artist;
            }

            set
            {
                artist = value;
            }
        }

        public ShowDTO(int id, string location, DateTime date,
                       int ticketsAvailable, int ticketsSold,
                       ArtistDTO artist)
        {
            this.Id = id;
            this.Location = location;
            this.Date = date;
            this.TicketsAvailable = ticketsAvailable;
            this.TicketsSold = ticketsSold;
            this.Artist = artist;
        }
    }
}
